hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOzoneManager.java

```diff
@@ -486,6 +486,7 @@ public void testDeleteNonEmptyBucket() throws Exception {
     String dataString = RandomStringUtils.randomAscii(100);
     KeyArgs keyArgs = new KeyArgs(volumeName, bucketName, keyName, userArgs);
     keyArgs.setSize(100);
+    keyArgs.setUserName(userName);
```

Comment:
why do we need to explicitly set UserName here? should it be derived from RPC layer on server with getRemoteUser()?

Reply:
yes, but here we are directly using OM instance.